source/whats-new.txt

```diff
@@ -0,0 +1,34 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+
+* :ref:`Version 2.19 <version-2.19>`
+
+.. _upcoming-breaking-changes:
+
+.. _version-2.19:
+
+What's New in 2.19 
+-------------------
+
+New features of the 2.19 driver release include:
+
+- The ``ObjectSerializer`` allows deserialization of only safe types, as determined by 
+  the configurable ``AllowedTypes`` function. 
+- Changed the default ``LinqProvider`` to LINQ3.
+- Added builders for creating Atlas searches.
+- Added support for Range Indexes preview.
```

Comment:
[question] Should "Range Indexes" be capitalized? I'm not seeing it as a formal feature name.

Reply:
I went through the same process. It's capitalized in all relevant work tickets but not in our docs, so I'll de-capitalize here.